spec/controllers/signatures_controller_spec.rb

```diff
@@ -86,6 +86,17 @@
         end
       end
 
+      it 'doesn\'t send email notification to the petition creator if the petition is already in moderation' do
+        petition_in_moderation = FactoryGirl.create(:petition, state: 'sponsored')
+        sponsor = FactoryGirl.create(:sponsor, petition: petition_in_moderation)
+        signature = sponsor.create_signature(FactoryGirl.attributes_for(:pending_signature, petition: petition_in_moderation))
+        perform_enqueued_jobs do
```

Comment:
We could probably use another approach to assert that the request does not add any jobs to the queue:  http://edgeapi.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-assert_no_enqueued_jobs
```
assert_no_performed_jobs do
  get :verify, :id => signature.id, :token => signature.perishable_token
end
```